From 4117c536c678c75683236e48246d78aa87a6212d Mon Sep 17 00:00:00 2001
From: Kevin Wolf <kwolf@redhat.com>
Date: Thu, 23 Oct 2014 10:10:04 +0200
Subject: [PATCH 01/19] libcacard: link against qemu-error.o for error_report()

Message-id: <1414059011-15516-2-git-send-email-kwolf@redhat.com>
Patchwork-id: 61835
O-Subject: [RHEL-7.1 qemu-kvm PATCH v2 1/8] libcacard: link against qemu-error.o for error_report()
Bugzilla: 1088176
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Max Reitz <mreitz@redhat.com>

From: Stefan Hajnoczi <stefanha@redhat.com>

Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
(cherry picked from commit 975a0015ee380f49a3be744279a6a06ab97e960a)
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>

Conflicts:
	libcacard/Makefile

Signed-off-by: Kevin Wolf <kwolf@redhat.com>
---
 libcacard/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/libcacard/Makefile b/libcacard/Makefile
index 6bdf0c4..c93cc6a 100644
--- a/libcacard/Makefile
+++ b/libcacard/Makefile
@@ -4,7 +4,8 @@ TOOLS += vscclient$(EXESUF)
 
 # objects linked into a shared library, built with libtool with -fPIC if required
 libcacard-obj-y = $(stub-obj-y) $(libcacard-y)
-libcacard-obj-y += util/osdep.o util/iov.o util/cutils.o util/qemu-timer-common.o util/error.o util/hexdump.o
+libcacard-obj-y += util/osdep.o util/iov.o util/cutils.o util/qemu-timer-common.o util/hexdump.o
+libcacard-obj-y += util/error.o util/qemu-error.o
 libcacard-obj-$(CONFIG_WIN32) += util/oslib-win32.o util/qemu-thread-win32.o
 libcacard-obj-$(CONFIG_POSIX) += util/oslib-posix.o util/qemu-thread-posix.o
 libcacard-obj-y += $(filter trace/%, $(util-obj-y))
-- 
1.8.3.1

